smt/surrogate_models/krg_based.py

```diff
@@ -1082,18 +1090,23 @@ def grad_minus_reduced_likelihood_function(log10t):
                     )
 
                     for i in range(len(self.noise0)):
+                        v= i + len(self.theta0)
+                        v2=deepcopy(v)
```

Comment:
v2 is an pointer of int but these pointers arise problems with multistart when calling the constraint.

I have not investigating the source of the problem since the deepcopy was a good go-around.

Reply:
Ok, I will see that, I've problems with the new implementation